src/shared/copy/notifications/categories/accounts-users-orgs.ts

```diff
@@ -44,6 +45,17 @@ export const accountRenameSuccess = (
   message: `Account "${oldAccountName}" was successfully renamed to "${newAccountName}"`,
 })
 
+export const deleteAccountWarning = (
+  buttonElement: NotificationButtonElement
+): Notification => ({
+  ...defaultWarningNotification,
+  message: `All additional users must be removed from the current organization before this account can be deleted.\n`,
```

Comment:
why the newline here?

Reply:
The newline is to ensure that the button element (which is an anchor tag) appears on the second line of the notification (instead of extending the notification to the right.)